aspnetcore/fundamentals/host/hosted-services.md

```diff
@@ -94,14 +94,20 @@ The <xref:Microsoft.Extensions.Hosting.IHostedService> interface defines two met
 
 The hosted service is activated once at app startup and gracefully shut down at app shutdown. If an error is thrown during background task execution, `Dispose` should be called even if `StopAsync` isn't called.
 
-## BackgroundService
+## BackgroundService base class
 
-`BackgroundService` is a base class for implementing a long running <xref:Microsoft.Extensions.Hosting.IHostedService>. `BackgroundService` provides the `ExecuteAsync(CancellationToken stoppingToken)` abstract method to contain the service's logic. The `stoppingToken` is triggered when [IHostedService.StopAsync](xref:Microsoft.Extensions.Hosting.IHostedService.StopAsync*) is called. The implementation of this method returns a `Task` that represents the entire lifetime of the background service.
+<xref:Microsoft.Extensions.Hosting.BackgroundService> is a base class for implementing a long running <xref:Microsoft.Extensions.Hosting.IHostedService>. `BackgroundService` provides the [ExecuteAsync(CancellationToken)](xref:Microsoft.Extensions.Hosting.BackgroundService.ExecuteAsync*) abstract method to contain the service's logic. The cancellation token is triggered when [IHostedService.StopAsync](xref:Microsoft.Extensions.Hosting.IHostedService.StopAsync*) is called.
 
-In addition, *optionally* override the methods defined on `IHostedService` to run startup and shutdown code for your service:
+[StartAsync(CancellationToken)](xref:Microsoft.Extensions.Hosting.BackgroundService.StartAsync*) is called to start the background service. The cancellation token is signalled if the startup process is interrupted. The implementation returns a `Task` that represents the startup process for the service.
```

Comment:
I know you're just reformatting this part, but the details about StartAsync and StopAsync don't belong here, they belong in the description for IHostedService. Someone using BackgroundService should only be using ExecuteAsync and that's what needs coverage in this section.

E.g. 
ExecuteAsync(CancellationToken) is called to start the background service. The implementation returns a Task that represents the entire lifetime of the background service. No further services are started until ExecuteAsync becomes asynchronous (link to eng issue), such as by calling `await` so avoid doing long, blocking initialization work here.